File: OmkarFCUBEAPP/src/app/models/dieselstatementrptlistmodel.ts
import { Paginationmodel } from "./paginationmodel";
import { Dieselstatementrptmodel } from "./dieselstatementrptmodel";

export class Dieselstatementrptlistmodel {
    dieselStatementRptlist: Dieselstatementrptmodel[] = [];
    pageMetaData: Paginationmodel = new Paginationmodel;
}

File: FCUBEAPI/FleetTrans/Models/DieselStatementRptListModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FleetTrans.Models
{
    public class DieselStatementRptListModel
    {
        public List<DieselStatementRptModel> DieselStatementRptlist { get; set; }
        public PaginationMetaData PageMetaData { get; set; }
    }
}
